src/Components/Milestone.css

```diff
@@ -12,6 +12,7 @@
 .p-timeline .p-timeline-event-content,
 .p-timeline .p-timeline-event-opposite {
   line-height: 1;
+  border-radius: 9%;
```

Comment:
**Remove this line from the `Milestone.css` file.**

We already had another PR #1485 to add the `border-radius` to the Milestone components! 

```suggestion
```